Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -79,6 +80,28 @@ extension Message {
     }
   }
 
+  /// Creates a new message by decoding the given string containing a
+  /// serialized message in JSON format.
+  ///
+  /// - Parameter jsonString: The JSON-formatted string to decode.
+  /// - Parameter extensions: An ExtensionMap
+  /// - Parameter options: The JSONDecodingOptions to use.
+  /// - Throws: `JSONDecodingError` if decoding fails.
+  public init(
+    jsonString: String,
+    extensions: ExtensionMap?,
```

Comment:
You could just default this and have one init (still source compat), and it means the decoder doesn't have to make it optional.

Reply:
You're right.  We don't promise binary/ABI compat, so there's no compelling reason to not take advantage of this simplification.